graylog2-web-interface/src/views/components/contexts/DefaultFieldTypesProvider.tsx

```diff
@@ -32,12 +33,19 @@ const defaultTimeRange: RelativeTimeRange = { type: 'relative', from: 300 };
 const DefaultFieldTypesProvider = ({ children }: { children: React.ReactElement }) => {
   const currentQuery = useCurrentQuery();
   const currentStreams = useMemo(() => filtersToStreamSet(currentQuery?.filter).toArray(), [currentQuery?.filter]);
-  const { data: currentFieldTypes } = useFieldTypes(currentStreams, currentQuery?.timerange || defaultTimeRange);
-  const { data: allFieldTypes } = useFieldTypes([], currentQuery?.timerange || defaultTimeRange);
+  const { data: currentFieldTypes, refetch: refreshCurrentTypes } = useFieldTypes(currentStreams, currentQuery?.timerange || defaultTimeRange);
```

Comment:
Not necessarily related to this PR, but can't we disable the fetching in this case, if `currentStreams` is empty?

Reply:
That should be handled by `react-query`'s caching, as in this case the generated query key should be identical for both calls.

Reply:
Alright!